lib/engines/hckinstall/hckinstall.rb

```diff
@@ -363,9 +367,10 @@ def prepare_studio_drives
     def copy_drivers
       @logger.info('HCKInstall: Copy all drivers')
       FileUtils.rm_rf("#{@hck_setup_scripts_path}/drivers")
+      # (src, dest, preserve = false, dereference_root = false, remove_destination = false);
       FileUtils.copy_entry(@project.options.install.driver_path,
                            "#{@hck_setup_scripts_path}/drivers",
-                           remove_destination: true)
+                           false, false, true)
```

Comment:
First bug spotted by Sorbet :+1:

By the way, why would we want `remove_destination` when we remove the whole directory just before that?

Reply:
Opened: https://github.com/ruby/ruby/pull/10081